identity/authorization.go

```diff
@@ -73,10 +73,10 @@ func (c *Client) IsAuthorizedByStringExternalID(
 	opts ...grpc.CallOption,
 ) (*identitypb.IsAuthorizedResponse, error) {
 	return c.IsAuthorizedWithRawRequest(ctx, &identitypb.IsAuthorizedRequest{
-		Subject: &identitypb.DigitalTwinIdentifier{Filter: &identitypb.DigitalTwinIdentifier_Property{
-			Property: &identitypb.Property{
-				Definition: &identitypb.PropertyDefinition{Property: externalIDProperty},
-				Value:      &identitypb.Property_ObjectValue{ObjectValue: objects.String(externalID)},
+		Subject: &identitypb.DigitalTwinIdentifier{Filter: &identitypb.DigitalTwinIdentifier_PropertyFilter{
+			PropertyFilter: &identitypb.PropertyFilter{
+				Type:  externalIDProperty,
+				Value: objects.String(externalID),
```

Comment:
I think this will be affected by the breaking changes @bakkelim refers to [here](https://github.com/indykite/jarvis-proto/pull/2004)

Reply:
This is correct. The proto files in buf.build has already been updated with my changes, so the client side here has been created with updated changes.